app/routes.js

```diff
@@ -51,10 +51,8 @@ router.post('/organisations/:orgId/adduser', async (req, res) => {
   const response = await postAddUser({ orgId, data: req.body });
 
   if (response.success) {
-    res.send('Confirmation page');
-    // TODO: Uncomment when confirmation page is done
-    // return res.redirect('/organisations/:orgId/adduser/success (TBD)');
-  }
+    res.redirect(`/organisations/${orgId}/adduser/confirmation`);
+  } else res.send('Error adding user');
```

Comment:
do we need the else? should shortcut when we have success

Reply:
linting complains there is nothing returned